### DISCUSSION TITLE
Audio-over-IP

### DISCUSSION THREAD

**Discussion by krono-i2** [ExistentialAudio/BlackHole#127] [Feature Requests]
Hi guys,
what do you think about integration of blackhole to AoIP tool OpenOB?

**Comment by devinroth** [comment]
That would be cool.

**Comment by krono-i2** [comment]
I found out gstreamer is not supported anymore by the last versions of macos.
We should consider to implement a custom solution based on RTP.

**Comment by datadata95** [comment]
Maybe could be done with the help of including this project or the use of WebRTC in some way? https://github.com/eugenehp/trx
